src/Microsoft.SqlTools.ServiceLayer/Scripting/ScriptingService.cs

```diff
@@ -133,14 +132,21 @@ public void InitializeService(ServiceHost serviceHost)
         {
             try
             {
+                ConnectionInfo connInfo;
+                ScriptingService.ConnectionServiceInstance.TryFindConnection(
+                    parameters.ConnectionString,
+                    out connInfo);
+                if (connInfo != null)
+                {
+                    parameters.ConnectionString = ConnectionService.BuildConnectionString(connInfo.ConnectionDetails);
```

Comment:
>parameters.ConnectionString = ConnectionService.BuildConnectionString(connInfo.ConnectionDetails); [](start = 20, length = 98)

could you add a comment why we need to rebuild the connection string?

Reply:
Done